src/unstract/sdk/__init__.py

```diff
@@ -1,4 +1,4 @@
-__version__ = "0.27.1"
+__version__ = "0.28.0"
```

Comment:
@chandrasekharan-zipstack [MINOR] These changes need a minor version bump

Reply:
@harini-venkataraman since we've made a minor version bump with a dependency, we require the SDK to also be a minor version bump - we've been following this as a practice I believe.
CC: @gaya3-zipstack @hari-kuriakose your thoughts on this?

Reply:
@chandrasekharan-zipstack There's a minor version update for a dependency, hence a minor version bump is required here too. Agree.